### PR TITLE
Deploy changes

### PR DESCRIPTION
**Pull request by zie1ony**
### Overview

Make deploy data accessible.

### CODE REVIEW

File: packages/sdk/src/lib/CLValue.ts [hunk 1]
```diff
@@ -358,25 +346,12 @@ class U256 extends NumberCoder {
   }
 
   public static fromBytes(bytes: ByteArray): Result<U128> {
```
Comment (George-cl) [inline packages/sdk/src/lib/CLValue.ts]: Shouldn't this be `Result<U256>`?

File: packages/ui/package.json [hunk 2]
```diff
@@ -21,7 +21,7 @@
     "@types/react-router-dom": "^5.0.1",
     "bootstrap": "^4.3.1",
     "bootstrap-components": "^1.1.287",
-    "casper-client-sdk": "^1.0.9",
+    "casper-client-sdk": "^1.0.10",
```
Comment (George-cl) [inline packages/ui/package.json]: Should this be updated to ^1.0.11?

**Review by George-cl — APPROVED** [review]
All looks good to me - left a few comments on small things - nice work!

File: packages/sdk/src/lib/CLValue.ts (hunk 1 above)
Comment (zie1ony) [reply packages/sdk/src/lib/CLValue.ts]: good catch

File: packages/ui/package.json (hunk 2 above)
Comment (zie1ony) [reply packages/ui/package.json]: I have changed it to use local sdk from the repo.

**Pull request merged**